src/ui/server/constants/templates/insurance/eligibility/index.ts

```diff
@@ -2,19 +2,15 @@ const COMPANIES_HOUSE_EXIT_PAGE = 'insurance/eligibility/companies-house-exit.nj
 
 export const ELIGIBILITY_TEMPLATES = {
   CHECK_IF_ELIGIBLE: 'insurance/eligibility/check-if-eligible.njk',
-  UK_GOODS_OR_SERVICES: 'insurance/eligibility/uk-goods-or-services.njk',
   TOTAL_VALUE_INSURED: 'insurance/eligibility/total-value-insured.njk',
   COVER_PERIOD: 'insurance/eligibility/cover-period.njk',
   LONG_TERM_COVER: 'insurance/eligibility/long-term-cover.njk',
-  COMPANIES_HOUSE_NUMBER: 'insurance/eligibility/companies-house-number.njk',
   NO_COMPANIES_HOUSE_NUMBER: COMPANIES_HOUSE_EXIT_PAGE,
   COMPANIES_HOUSE_SEARCH: 'insurance/eligibility/companies-house-search.njk',
   COMPANIES_HOUSE_UNAVAILABLE: 'insurance/eligibility/companies-house-unavailable.njk',
   COMPANY_NOT_ACTIVE: COMPANIES_HOUSE_EXIT_PAGE,
   COMPANY_DETAILS: 'insurance/eligibility/company-details.njk',
-  END_BUYER: 'insurance/eligibility/end-buyer.njk',
   CANNOT_APPLY_MULTIPLE_RISKS: 'insurance/eligibility/cannot-apply-multiple-risks.njk',
   CHECK_YOUR_ANSWERS: 'insurance/eligibility/check-your-answers.njk',
   ELIGIBLE_TO_APPLY_ONLINE: 'insurance/eligibility/eligible-to-apply-online.njk',
-  HAVE_AN_ACCOUNT: 'insurance/eligibility/have-an-account.njk',
```

Comment:
Are we now able to update any others? E.g `TOTAL_VALUE_INSURED,` COVER_PERIOD`

Reply:
Not yet, as they are not yes no radios

Reply:
Made a note of this, will create a ticket